pyproject.toml

```diff
@@ -21,3 +21,43 @@ exclude = '''
                      # the root of the project
 )
 '''
+
+[tool.poetry]
+name = "pinecone"
+version = "2.2.2"
+description = "The official Python client for Pinecone's vector database"
+authors = ["Zachary Proser <zack.p@pinecone.io>"]
```

Comment:
@jamescalam I'm thinking this should be `support@pinecone.io` - what do you think?